base_layer/core/src/base_node/state_machine_service/states/block_sync.rs

```diff
@@ -74,7 +74,7 @@ impl BlockSync {
             ));
 
             let _ = status_event_sender.broadcast(StatusInfo {
-                bootstrapped: false,
+                bootstrapped: true,
```

Comment:
This should not be false (same for in the HeaderSync state) but just setting this to true nullifies the whole purpose of the Bootstrapped flag. The current value should be drawn from the Shared state value `shared.bootstrapped_sync`

Reply:
I think this comes down to, should header sync change this value?

Reply:
nope, only the first time you get to Listening should change it